ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/router-bgp-evpn.cfg

```diff
@@ -143,6 +143,8 @@ router bgp 65101
    vrf TENANT_A_PROJECT01
       rd 192.168.255.3:11
       evpn multicast
+         gateway dr election algorithm preference 10000
+         !
```

Comment:
Should we print this ! even if no other command is printed?

Reply:
Removed. Thanks.